src/Symfony/Component/Messenger/Transport/Doctrine/Connection.php

```diff
@@ -52,12 +53,14 @@ class Connection
     private $configuration = [];
     private $driverConnection;
     private $schemaSynchronizer;
+    private $schemaConfig;
 
-    public function __construct(array $configuration, DBALConnection $driverConnection, SchemaSynchronizer $schemaSynchronizer = null)
+    public function __construct(array $configuration, DBALConnection $driverConnection, SchemaSynchronizer $schemaSynchronizer = null, SchemaConfig $schemaConfig = null)
```

Comment:
when is the argument passed? shouldn't we adjust some configuration in the FrameworkBundle?

Reply:
I'm wondering if the argument is usefull. Actually I retrieve the SchemaConfig from the driverConnection.

Maybe we don't need this argument at all. wdyt ?

Reply:
I'd remove it indeed